store/sqlstore/post_store.go

```diff
@@ -2272,6 +2272,23 @@ func (s *SqlPostStore) AnalyticsPostCount(options *model.PostCountOptions) (int6
 		query = query.Where(sq.Eq{"p.DeleteAt": 0})
 	}
 
+	if options.ExcludeSystemPosts {
+		query = query.Where("p.Type NOT LIKE 'system_%'")
+	}
+
+	if options.SinceUpdateAt > 0 {
+		subQuery := sq.Or{sq.Gt{"p.UpdateAt": options.SinceUpdateAt}}
+
+		if options.SincePostID != "" {
```

Comment:
I am wondering if it's more correct to remove this condition. Even if we don't supply the postID, we'd want to include posts from the timestamp as well right? So if sincePostID is empty, it will just return all posts with the timestamp, which should be fine.

For example, there is a setting `MessageExportSettings.ExportFromTimestamp`. If a user sets that and kicks off a job, then the expectation would be to export from that timestamp _including_ it. In that case, there would be no `sincePostID` and we would be excluding the posts with the timestamp which does not feel like correct behavior.